schemas/org/openstax/ec/common_schema_types.rb

```diff
@@ -0,0 +1 @@
+fixed :uuid, 16
```

Comment:
when creating dsls...do we really want to import the whole enchilada of common types into there?

Reply:
good point - we could do `import 'types/uuid'`, `import 'types/timestamp1'`, etc